metrics.nim

```diff
@@ -17,25 +17,33 @@ when defined(metricsTest):
 else:
   {.pragma: testOnly, deprecated: "slow helpers used for tests only".}
 
-import std/[locks, monotimes, os, sets, tables, times]
+import std/[locks, monotimes, os, sets, times], metrics/shseq
+
+export shseq
 
 when defined(metrics):
   import std/[algorithm, hashes, strutils, sequtils], stew/ptrops, metrics/common
 
-  export tables # for custom collectors that need to work with the "Metrics" type
-
 type
-  LabelKey = object
-    # Helper type to work around the lack of heterogenous key support in `Table`
-    data: seq[string]
-    refs: ptr UncheckedArray[string]
-    refslen: int
+  CStringArray = object
```

Comment:
I think its unsafe to call type like this, because there present internal Nim type `cstringArray`.

Reply:
[6674364](https://github.com/status-im/nim-metrics/pull/97/commits/66743644b641b991d3410e4902a6161c2984d931)

technically not a problem because of the initial capital `C` but renamed to avoid confusion